examples/sphere.py

```diff
@@ -294,11 +368,15 @@ def sphere_main(algorithm,
                 # Record and display metrics.
                 metrics["QD Score"]["x"].append(itr)
                 metrics["QD Score"]["y"].append(archive.stats.qd_score)
+                metrics["Mean QD Score"]["x"].append(itr)
```

Comment:
You called it "Normalized QD Score" above -- I actually don't think we need this though.